cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/ClusterBootstrap.scala

```diff
@@ -37,7 +37,7 @@ final class ClusterBootstrap(implicit system: ExtendedActorSystem) extends Exten
 
   AkkaVersion.require("cluster-bootstrap", "2.5.19")
 
-  val settings = ClusterBootstrapSettings(system.settings.config, log)
+  val settings: ClusterBootstrapSettings = ClusterBootstrapSettings(system.settings.config, log)
```

Comment:
private?

Reply:
I think we typically have settings classes as public